### DISCUSSION TITLE
Nes Classic Mini diagram

### DISCUSSION THREAD

**Discussion by fermel01** [darthcloud/BlueRetro#319] [Q&A]
What is the wiring diagram for a Nintendo NES / SNES Classic to an ESP32?

Thanks in advance

**Comment by darthcloud** [comment]
See https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-SNES-Mini-Internal-Install

for instructions. Look at picture for solder point and look at table for ESP IO. I don't have a NES mini so can't provide instructions.

**Reply by kleffa** [reply]
In the cable instruction I missed the table with pinout to esp32

**Reply by jane-remover** [reply]
@darthcloud could you clarify the following (regarding https://github.com/darthcloud/BlueRetro/wiki/img/mini/esp_mount.png)?

I tried to make sense of the pin table, but there are some outliers which I'm finding confusing. A schematic could help make this clearer.

SVN: Why are the IO5,25,26,27 2K pull up resistors being pulled up to SVN? The instructions say to pull up to 3V3.
IO0: What is the pull up resistor to 3V3 here for? What is the wire going to IO0 for?
IO2: Why is only 1 wire going to IO2? I was expecting there to be 2 wires, both P1 and P2 "detect to mini" connected here?

**Reply by darthcloud** [reply]
The mini internal wiring will change for v1.8, so its better to wait for an update.

**Reply by jane-remover** [reply]
thanks for the heads up! roughly when will v1.8 be released?

**Reply by kleffa** [reply]
> The mini internal wiring will change for v1.8, so its better to wait for an update.

Please include the table for external adapters, since it also is compatible with Wiimote's port

**Reply by jane-remover** [reply]
@darthcloud any update on the wiring guide?

**Reply by darthcloud** [reply]
It's on the ice indefinitely unfortunately.

I may return to it sometimes this summer, right now I'm trying to fix Switch BT issues.
